schemas/number.json

```diff
@@ -198,6 +223,10 @@
       "type": "number",
       "description": "Meter is the SI unit for length. This type is used for rare cases in which the option of an uncertainty is not desired."
     },
+    "squareMeter": {
+      "type": "number",
```

Comment:
"squareMeter" should be a nonNegativeNumber . You can decide if you want to allow the definition of an uncertainty or not.